helm/ray/templates/raycluster-storage.yaml

```diff
@@ -5,8 +5,8 @@ metadata:
   namespace: {{ .Release.Namespace }}
 spec:
   accessModes:
-  - ReadWriteMany
+  - {{ .Values.storage.accessMode | default "ReadWriteMany" }}
   resources:
     requests:
       storage: 20Gi
-  storageClassName: {{ .Values.storage.storageClassName | default "default" }}
\ No newline at end of file
+  storageClassName: {{ .Values.storage.storageClassName | default "default" }}
```

Comment:
Then if that's the case we will need to make this more declarative by configuring it through values.yaml, and as an entrypoint, via cli. We will need to ask that question to user whether they want to do it manually, then I'd say we don't put `storageClassName` if that's case, we need to ask user to provide `claim ref`  and we will need to add some lines like below in PVC to match th emanually craeted PV

```
  selector:
    matchLabels:
      pv-label-key: pv-label-value
```

this will match the manually created PV.

But I'd suggest us to move on with the current structure by not making significant changes, And once it is done with basics, we could move on with customs and adjust our setup accordingly.

WDYT?

Reply:
made some changes on PVC setup,could you pls take a look at it?

Reply:
That looks fine to me btw! I think that should cover that situation correctly